README.md

```diff
@@ -124,8 +129,16 @@ by adding -DUSE_CPU_ONLY=1 to the cmake command line.
     cd leela-zero
     git submodule update --init --recursive
 
-    # Install build depedencies
-    brew install boost cmake zlib
+    # Install leelaz build depedencies
+    brew install boost cmake zlib curl
+    
+    # Optional : if you want to contribute to the distributed effort, you need these dependencies as well
+    brew install qt5
+    # after qt5 install, then you also need to do the post install for mac, replace 5.x.x with your qt5 version
+    export QT5DIR=/usr/local/Cellar/qt/5.x.x
```

Comment:
None of this should be needed aside from fixing the path.

Reply:
@gcp 
i dont understand your explanation : 
- so only path export may be needed for mac ?
- qt is factory installed on all macs, or somehow shipped as a dependency while installing other needed packages ?
- to sum up, only lines 137 and 138 are needed then ?

thanks